src/automations/JobPost.ts

```diff
@@ -197,7 +201,31 @@ export default class JobPost {
             question.answer_type_key = question.answer_type.key;
             delete question.answer_type;
             question.id = null;
+            if (
+                question["question_options"] &&
+                question["question_options"].length > 0
+            ) {
```

Comment:
I believe this can be written
```
if (question?.question_options.length > 0) {...}
``` 
But feel free to leave it as it is if you find it more explicit.

Reply:
Done :+1: